DoublyLinkedList/header.h

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
+using namespace std;
+// Node Creation
+class Node{                                                     //class
+        int data;
+        Node* next;
+        Node* prev;
+        public:                                                 //Access specifier
+
+        void push(Node** head_ref,int value);                   // Insertion of value at front
```

Comment:
why is Node** an argument ? push just pushed value to front. why should Node be passed ?

Reply:
In order to depict different method of handling the node I used passing address in one method and passing node in another method sir. Both is comfortable to execute, if I need to change, I will change sir